bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -26,6 +26,8 @@ public void testConnectToListenerByWorkerAddr() {
         UcpEndpoint endpoint = worker.newEndpoint(epParams);
         assertNotNull(endpoint.getNativeId());
 
+        worker.flush();
+
```

Comment:
why do we need flush in Java? it's using UCP layer, which should already do flush in ucp_ep_close_nb()